units/host_allocator.go

```diff
@@ -230,7 +231,7 @@ func (j *hostAllocatorJob) Run(ctx context.Context) {
 		timeToEmptyNoSpawns = time.Duration(0)
 	} else {
 		hostsAvailNoSpawns := hostsAvail - correctedHostsSpawned
-		maxHours := 2532000
+		const maxHours = 2532000
```

Comment:
We don't usually define constants in function; can you move the constants to the top of the file? (I find this helpful for referencing across functions if necessary, and to maintain the idea that the value _is_ a constant)

Reply:
For this constant also, a comment about where the number came from would be helpful

Reply:
I think it makes sense to define them in-function because we only use them in-function, and given that they're one-time-use it's more legible to have them defined close to where they're used. This is in contrast to other constants which are defined at the top and used all over the place, for example, `hostAllocatorJobName`.